ANMN/burst_averaged_product/generate_nc_file_att

```diff
@@ -0,0 +1,227 @@
+[global_attributes]
+Conventions             = CF-1.6,IMOS-1.3
+acknowledgement         = Any users of IMOS data are required to clearly acknowledge the source of the material derived from IMOS in the format: "Data was sourced from the Integrated Marine Observing System (IMOS) - IMOS is a national collaborative research infrastructure, supported by the Australian Government." If relevant, also credit other organisations involved in collection of this particular datastream (as listed in "credit" in the metadata record).
+author_email            = laurent.besnard@utas.edu.au
+author_name             = Besnard, Laurent
+citation                = The citation in a list of references is: "IMOS [year-of-data-download], [Title], [data-access-URL], accessed [date-of-access].".
+comment                 = Geospatial vertical min/max information has been computed using the Gibbs-SeaWater toolbox (TEOS-10) v3.02 from latitude and relative pressure measurements (calibration offset usually performed to balance current atmospheric pressure and acute sensor precision at a deployed depth).
```

Comment:
Don't hard-code `comment`. Copy what's in the input file.
Same for most of the other attributes here, especially things that can vary from file to file:
`institution`
`institution_references`
`references`
variable `comment` attributes